### PR TITLE
adonis-remember-token cookie path issue fix.

### PR DESCRIPTION
**Pull request by 9636565**

### CODE REVIEW

File: src/Schemes/Session.js [hunk 1]
```diff
@@ -80,7 +80,10 @@ class SessionScheme extends BaseScheme {
      */
     if (rememberToken && duration) {
       this._ctx.response.cookie(this.rememberTokenKey, rememberToken, {
-        expires: new Date(Date.now() + duration)
+        expires: new Date(Date.now() + duration),
+        httpOnly: true,
+        sameSite: true,
+        path: '/'
```
Comment (thetutlage) [inline src/Schemes/Session.js]: May I know, whey we are hardcoding `path` here and not reading it from the config file?

Comment (9636565) [reply src/Schemes/Session.js]: this commit 39deaf0 did not pass the tests. So I hardcodded them.

File: src/Auth/index.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ class Auth {
       return this._authenticatorsPool[name]
     }
 
-    const config = this.Config.get(`auth.${name}`)
+    const config = _.assign(this.Config.get(`auth.${name}`), this.Config.get(`session.cookie`))
```
Comment (RomainLanz) [inline src/Auth/index.js]: We shouldn't hardcode the merge of the `session.cookie` configuration.

**Comment by thetutlage** [comment]
@9636565 The changes looks fine to me. Can you also add some tests around the change? Lemme know if you need some help

**Pull request closed without merging**